magefile.go

```diff
@@ -202,3 +210,16 @@ func hasChangesIn(prevVersion semver.Version, version semver.Version, folder str
 
 	return false, err
 }
+
+func getLdflags() (string, error) {
+	if ldflags := os.Getenv("LDFLAGS"); ldflags != "" {
```

Comment:
💭 considering we're already parsing and re-routing these vars, perhaps it's more intuitive / ergonomic to give ourself a special env var for setting captain version, like `$CAPTAIN_VERSION` ?